src/options.cpp

```diff
@@ -1320,6 +1322,31 @@ void show_options(bool ingame)
             g->init_ui();
             if( ingame ) {
                 g->refresh_all();
+
+                DebugLog( D_INFO, DC_ALL ) << "Loaded tileset: " << OPTIONS["TILES"].getValue();
+
+                tile_loading_report(termap, "Terrain", "");
+                tile_loading_report(furnmap, "Furniture", "");
+                tile_loading_report(Item_factory::Item_factory().get_all_itypes(), "Items", "");
+                tile_loading_report(MonsterGenerator::generator().get_all_mtypes(), "Monsters", "");
```

Comment:
I'm guessing someone else who is using that function might want a copy. I'll leave this alone, since it's only once. I wish there was a global instance of this map like furnmap and termap.